crates/rattler_solve/src/lib.rs

```diff
@@ -70,6 +70,22 @@ impl fmt::Display for SolveError {
     }
 }
 
+/// Represents the channel priority option to use during solves.
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub enum ChannelPriority {
+    /// The channel that the package is first found in will be used as the only channel
+    /// for that package.
+    Strict,
+
+    /// Conda also has "Flexible" as an option, where packages present in multiple channels
```

Comment:
maybe we shoudl use `//` so that these comments do not show up in teh documentation for a feature we don't implement.

Reply:
Yeah sounds good, I only did that cuz with `//` it formats the comments right against the disabled comment, but likely more confusing for it to be in the documentation